tests/get_paths.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Common variables used by all scripts
+stackDirs=''
+stacksPath=''
+POSITIONAL_ARGS=()
+
+# Function to parse all arguments
+parse_arguments() {
+    while [[ $# -gt 0 ]]; do
+        case $1 in
+            --stackDirs)
+                stackDirs=$2
+                shift # past argument
+                shift
+                ;;
+            --stacksPath)
+                stacksPath=$2
+                shift # past argument
+                shift
+                ;;
+            -*|--*)
+                # Try script-specific handler if it exists
+                local consumed=0
+                if declare -f handle_additional_args > /dev/null 2>&1; then
+                    handle_additional_args "$@"
+                    consumed=$?
+                fi
+                
+                if [ $consumed -gt 0 ]; then
+                    # Script handler consumed some arguments
+                    for ((i=0; i<consumed; i++)); do
+                        shift
+                    done
+                else
+                    echo "Unknown option $1"
+                    exit 1
+                fi
+                ;;
+            *)
+                POSITIONAL_ARGS+=("$1") # save positional arg
+                shift # past argument
+                ;;
+        esac
+    done
+    
+    # Restore positional parameters
+    restore_positional_args POSITIONAL_ARGS
+}
+
+# Function to set default values for stack arguments
+set_stack_defaults() {
+    if [ -z "$stackDirs" ]; then
+        stackDirs=$(bash "$(pwd)/tests/get_stacks.sh")
+    fi
+
+    if [ -z "$stacksPath" ]; then
+        stacksPath="$(pwd)/stacks"
+    fi
+}
+
+# Helper function to restore positional parameters
+restore_positional_args() {
+    local -n args_array=$1
+    if [ ${#args_array[@]} -gt 0 ]; then
+        set -- "${args_array[@]}"
+    else
+        set --
+    fi
+} 
```

Comment:
You'll need to reset the script arguments under the script scope, does not work in a helper function.

```suggestion
```